Makefile

```
@@ -504,6 +499,59 @@ stats-download-kpi: ## Generate download KPI JSON for all published articles (op
+		echo "Usage: make solr-index docid=N | sqlwhere=CLAUSE | file=PATH [sync=1] [priority=N]"; \
+		exit 1; \
+	fi
+	@$(DOCKER_COMPOSE) exec -u $(CNTR_APP_USER) -w $(CNTR_APP_DIR) $(CNTR_NAME_PHP) \
+		php scripts/console.php solr:index \
+		$(if $(docid),--docid=$(docid)) \
+		$(if $(sqlwhere),--sqlwhere=$(sqlwhere)) \
+		$(if $(file),--file=$(file)) \
+		$(if $(priority),--priority=$(priority)) \
+		$(if $(filter 1,$(sync)),--sync)
+
+solr-delete: ## Enqueue (or sync) a Solr deletion (requires docid=N or query=QUERY; optional: sync=1)
```

checkmake (0.3.2)

[warning] Target body for "solr-delete" exceeds allowed length of 5 lines (12).

(maxbodylength)